app/helpers/application_helper.rb

```diff
@@ -2,72 +2,49 @@
 
 module ApplicationHelper
   def page_title
-    "#{@title}#{' -' if @title} #{t('layout.title')}"
+    "#{@title}#{' -' if @title} #{t('layout.title')}" # rubocop:disable Rails/HelperInstanceVariable
```

Comment:
would really rather leave these as failing cops to be fixed. but i've hit approve anyways

Reply:
Yeah I get that. Issue #1004 tracks fixing these but I think there's enough benefit to having the whole codebase pass that it's worth merging.